ui/src/routes/guides/indicators.svelte

```diff
@@ -22,21 +22,21 @@
 </script>
 
 <svelte:head>
-	<title>{toolName}: Guide</title>
+	<title>{toolName}: Indicators Guide</title>
 	<meta
-		content='{toolName}: usage guide for this tool'
+		content='{toolName}: usage guides for this tool'
```

Comment:
Probably best being explicit and use:

```suggestion
		content='{toolName}: how to explore the indicators'
```